zrml/swaps/src/tests.rs

```diff
@@ -1569,10 +1642,63 @@ fn swap_exact_amount_out_exchanges_correct_values_with_cpmm(
     });
 }
 
+#[test]
+fn swap_exact_amount_out_exchanges_correct_values_with_cpmm_with_fees() {
+    ExtBuilder::default().build().execute_with(|| {
+        frame_system::Pallet::<Runtime>::set_block_number(1);
+        let asset_amount_out = _1;
+        let asset_amount_in = 11223344556; // 10101010100 / 0.9
+        let asset_bound = Some(_2);
+        let max_price = Some(_3);
+        frame_system::Pallet::<Runtime>::set_block_number(1);
+        ASSETS.iter().cloned().for_each(|asset| {
+            let _ = Currencies::deposit(asset, &ALICE, _25);
+            let _ = Currencies::deposit(asset, &BOB, _10000);
+        });
+        assert_ok!(Swaps::create_pool(
+            BOB,
+            ASSETS.iter().cloned().collect(),
+            ASSETS.last().unwrap().clone(),
+            0,
+            ScoringRule::CPMM,
+            Some(BASE / 10),
```

Comment:
Since the other numbers in the test are not based on `MaxSwapFee`, changing `MaxSwapFee` will break the test. I'd rather not use the constant for that reason, and it doesn't really matter that this is the maximum fee.